src/encoded/__init__.py

```diff
@@ -215,9 +216,14 @@ def main(global_config, **local_config):
     # registered.
     config.include('.upgrade')
 
-    # initialize sentry reporting
-    sentry_sdk.init("https://ce359da106854a07aa67aabee873601c@o427308.ingest.sentry.io/5373642",
-                    integrations=[PyramidIntegration()])
+    # initialize sentry reporting, split into "production" and "mastertest", do nothing in local/testing
+    current_env = settings.get('env.name', None)
+    if current_env in [FF_ENV_PRODUCTION_GREEN, FF_ENV_PRODUCTION_BLUE]:
+        sentry_sdk.init("https://0d46fafce1d04ea2bfbe11ff15ca896e@o427308.ingest.sentry.io/5379985",
```

Comment:
Is this blurring which server is running (and hence which software is running?) by using the same DSN for either of these two? Is that a good idea?

Reply:
I think this is acceptable for now, but will change it if we decide to pull the trigger on sentry.